frontend/public/components/storage/attach-storage.tsx

```diff
@@ -159,6 +178,19 @@ class AttachStorageForm extends React.Component<
     const resourceObj = _.cloneDeep(originalObj);
     const template = resourceObj.spec.template;
 
+    this.setState({ inProgress: true });
+    if (showCreatePvc === 'new') {
+      k8sCreate(PersistentVolumeClaimModel, pvcObj).then(
```

Comment:
We need to chain these requests so that we don't update the workload until the PVC is successfully created